### PR TITLE
Admins can suspend/unsuspend users

### LINKED ISSUE #43: Admins can manage users
# Requirments
- [ ] Admins can suspend and unsuspend users
- [ ] User should not have login/access to features if they are suspended
- [ ] Automatically email when a user is suspended or unsuspended

### PR DESCRIPTION
**Pull request by KA2010**

### CODE REVIEW

**Review by hunterwilkins2 — CHANGES_REQUESTED** [review]
Hey, job good job on your code it looks good! On line 16 change it to be `{ status: 'suspended' }` and on line 43 change it to `{ status: 'active' }`

File: user.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+router.patch('/suspendUser', auth.verifyAdmin, async (req, res, next) => {
+    try {
+        const { name } = req.body
+
+        const user = await User.findOne({ name })
+        if(user.userType === 'admin') 
+            throw Error('Cannot susbend an admin user')
+
+        const employee = await User.findOneAndUpdate({ name },
+            { userType: 'customer'},
```
Comment (KA2010) [inline user.js]: {status:'suspended'}

File: user.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+router.patch('/unsuspendUser', auth.verifyAdmin, async (req, res, next) => {
+    try {
+        const { name } = req.body
+        const employee = await User.findOne({ name })
+        if(employee.userType === 'admin') 
+            throw Error('Cannot unsusbend an admin user')
+
+        const user = await User.findOneAndUpdate({ name }, 
+            { userType: 'customer'},
```
Comment (KA2010) [inline user.js]: {status: 'active'}

Comment (hunterwilkins2) [reply user.js]: This needs to be changed in the actual code and then pushed back up to github. I can't change the code from a comment.

Comment (hunterwilkins2) [reply user.js]: Nvm, I actually can update the find from a comment. Thank you!

**Pull request merged**